validators/terravaloper183kevvgrqvav9sa0lwzhr54jngxw6sac90jvs2/README.md

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Could you add a header similar to the one from the [example profile](https://github.com/terra-money/validator-profiles/pull/437/files#diff-2db876394628438c041d08254ed57ba608099123cc68a350f10c5a42b54ec1d8R2-R4)

Reply:
updated